tests/AdminMenus/AdminReusableBlocksMenuCliTest.php

```diff
@@ -74,8 +99,8 @@
 
 	$key = 'shortdesc';
 
-	$this->assertIsArray($documentation);
-	$this->assertArrayHasKey($key, $documentation);
-	$this->assertArrayHasKey('synopsis', $documentation);
-	$this->assertSame('Generates reusable blocks admin menu class file.', $documentation[$key]);
+	expect($documentation)->toBeArray();
+	expect($documentation)->toHaveKey($key);
+	expect($documentation)->toHaveKey('synopsis');
+	expect($documentation[$key])->toBe('Generates reusable blocks admin menu class file.');
```

Comment:
Expectations are chainable, meaning you can do:

```suggestion
	expect($documentation)->toBeArray()
		->toHaveKey($key)
		->toHaveKey('synopsis');
		
	expect($documentation[$key])->toBe('Generates reusable blocks admin menu class file.');
```